core/src/types/operator/operator.rs

```diff
@@ -64,6 +66,8 @@ use crate::*;
 pub struct Operator {
     // accessor is what Operator delegates for
     accessor: Accessor,
+    // info stores the metadata for operator, users can use this metadata to get information of operator.
+    info: OnceCell<Arc<OperatorInfo>>,
```

Comment:
Hi, I think we no longer need to store `OperatorInfo` in the operator since building `OperatorInfo` is now cost-effective.

Reply:
ok